src/main.ts

```diff
@@ -20,7 +20,10 @@ async function main() {
       resizable: false,
     });
 
-    mainWindow.webContents.openDevTools();
+    // The following line is only needed if you want to use the dev tools
+    // mainWindow.webContents.openDevTools();
+    await createTray(mainWindow);
+    hideWindow(mainWindow);
 
     if (MAIN_WINDOW_VITE_DEV_SERVER_URL) {
```

Comment:
```suggestion
    if (MAIN_WINDOW_VITE_DEV_SERVER_URL) {
    mainWindow.webContents.openDevTools();
```
Ao inves de comentar o dev tools, mover ele pra dentro da execução de Dev.